File: playback/cast/src/fdroid/java/allen/town/podcast/playback/cast/CastEnabledActivity.java
package allen.town.podcast.playback.cast;


import android.view.Menu;

import allen.town.focus_common.activity.ToolbarBaseActivity;

/**
 * Activity that allows for showing the MediaRouter button whenever there's a cast device in the
 * network.
 */
public abstract class CastEnabledActivity extends ToolbarBaseActivity {
    public static final String TAG = "CastEnabledActivity";

    public final void requestCastButton(Menu menu) {
        // no-op
    }
}

File: playback/cast/src/free/java/allen/town/podcast/playback/cast/CastEnabledActivity.kt
package allen.town.podcast.playback.cast

import allen.town.focus_common.activity.ToolbarBaseActivity
import android.view.Menu

/**
 * Activity that allows for showing the MediaRouter button whenever there's a cast device in the
 * network.
 */
abstract class CastEnabledActivity : ToolbarBaseActivity() {
    fun requestCastButton(menu: Menu?) {
        // no-op
    }

    companion object {
        const val TAG: String = "CastEnabledActivity"
    }
}
